### ISSUE TITLE
Linking R-Instat to PICSA climate app?

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#5496]
Chris Clarke is working for the PICSA team and is updating his climate app.
It takes (annual) summary data of the sort we produce in R-Instat and displays the results very attractively and interactively.  For PICSA this is for mobile phones and is a sort of alternative option to having the graphs (Climatic > PICSA > Rainfall Graphs) on paper.
The app is [here](https://picsa-climate-tool.web.app).
 Here is an example, with data from Malawi.
![image](https://user-images.githubusercontent.com/11226469/64234576-888d2300-ceee-11e9-92ed-7ad0eb347774.png)

You can move the slider on the left - from 600 - and it will recalculate all the risks - which is what people currently do manually now.

@dannyparsons says it should be easy to link this app to R-Instat.  I think Chris would be positive about collaborating on such work.  It looks to me like an interesting @Patowhiz challenge, but needs discussion first about both how to do this and where it might fit into our priorities.  And that all depends on how large the task is.

**Comment by rdstern** [comment]
I have tentatively "promoted" this feature to Version 0.5.4, partly because I think it could be a useful "selling point" when demonstrating PICSA in Ethiopia at the end of October.  I hope with @dannyparsons and @Patowhiz this could perhaps be discussed when we are together in Cameroon?

**Comment by dannyparsons** [comment]
Discussed with Chris.

- Example data in app format here [makanjira.zip](https://github.com/africanmathsinitiative/R-Instat/files/3702059/makanjira.zip).
- For simplicity, we should export start and end dates in ISO Date format "2015-03-25"
- Blank for missing values
- Multi year seasons e.g. 2009-2010 should be written as 2009

**Comment by rdstern** [comment]
Does this mean that there should be a dialogue in **_Climatic > File > Export to PICSA App_**, because we will often want to export the data and then use the app separately.  Then there could simply be a button to perhaps launch the app from within R-Instat?  It could also go under a line in the PICSA menu.

**Comment by dannyparsons** [comment]
Yes we start with a new export dialog to create a csv file that can be loaded into the app. Closer integration can follow. I think there is some technical work needed to be done before this happens.

**Comment by Patowhiz** [comment]
I'm now commencing work on this integration

**Comment by lilyclements** [comment]
@Patowhiz how did it go when you looked into this?
@rdstern is this something we could dig into if we got an extension on the PICSA project?
